api/envoy/extensions/filters/http/tap/v3/tap.proto

```diff
@@ -26,4 +26,15 @@ message Tap {
   // Common configuration for the HTTP tap filter.
   common.tap.v3.CommonExtensionConfig common_config = 1
       [(validate.rules).message = {required: true}];
+
+  // Specific configuration except common config for the HTTP tap filter.
+  OutputConfig output_config = 2;
```

Comment:
I don't think this extra message adds much value here. This is already specific to HTTP. I would just inline the other option here.

Reply:
Ping. Please remove this extra message.

/wait